src/main/java/seedu/address/model/meeting/MeetingTime.java

```diff
@@ -39,6 +39,13 @@ public static boolean isValidMeetingTime(String test) {
         }
     }
 
+    /**
+     * Returns true if a given meeting has expired.
+     */
+    public boolean isExpiredMeetingTime() {
+        return this.dateTime.isBefore(LocalDateTime.now());
```

Comment:
I am confused, shouldn't the meeting time expire when the datetime is after the current time? Here the method runs true if the date time is before the current time.

Reply:
If the meeting is before now, it has expired right? If it after the current time, it should be in the upcoming meetings list.

Reply:
oh yeah, ok that clears things up